x/auth/mapper.go

```diff
@@ -50,10 +50,15 @@ func (am AccountMapper) NewAccount(ctx sdk.Context, acc Account) Account {
 	return acc
 }
 
+// Turn an address to key used to get it from the account store
+func AddressStoreKey(addr sdk.Address) []byte {
+	return append([]byte("account:"), addr.Bytes()...)
```

Comment:
Prefix can be a constant at the top of the file.

Reply:
Planning on switching the subspace kvstore system later anyways #1166